src/App.js

```diff
@@ -21,6 +39,14 @@ function App() {
       });
   }, []);
 
+  function deleteButtonHandler(id) {
+    const deletedItems = order.filter((deletedItem) => {
+      return id !== deletedItem.id;
+    });
+    setOrder(deletedItems);
+    console.log(deletedItems);
```

Comment:
I found it, the forbidden console.log XD

Reply:
HAHAHA i though maybe you wouldnt see it. I saw it after 😄